cmd/lakectl/cmd/local_install.go

```diff
@@ -0,0 +1,52 @@
+package cmd
+
+import (
+	"os"
+	"path"
+	"path/filepath"
+	"runtime"
+	"strings"
+
+	"github.com/spf13/cobra"
+)
+
+func currentExecutable() string {
+	ex, err := os.Executable()
+	if err != nil {
+		DieErr(err)
+	}
+	absolute, err := filepath.Abs(ex)
+	if err != nil {
+		DieErr(err)
+	}
+	return absolute
+}
+
+var localInstallGitPluginCmd = &cobra.Command{
+	Use:   "install-git-plugin <directory>",
+	Short: "set up `git data` (directory must exist and be in $PATH)",
```

Comment:
Can we add a long description here? I think in this case it's important to describe exactly how to use it

Reply:
Added!